components/eam/src/physics/cam/macrop_driver.F90

```diff
@@ -13,27 +13,14 @@ module macrop_driver
 
```

Comment:
As we are not using macrophysics after introducing CLUBB, Can we get rid of everything and remove macrop_driver?

Reply:
Removed macop_driver.F90